### PR TITLE
Future/peggy2 fix smartcontract tests

### PR DESCRIPTION
**Pull request by Brando753**

### CODE REVIEW

**Comment by smartyalgo** [comment]
For context, whats the reason to include those fields?

**Review by juniuszhou — APPROVED** [review]

File: x/ethbridge/types/msgs.go [hunk 1]
```diff
@@ -224,12 +224,12 @@ func ComputeProphecyID(cosmosSender string, sequence uint64, ethereumReceiver st
 	bytes, _ := arguments.Pack(
```
Comment (smartyalgo) [inline x/ethbridge/types/msgs.go:224]: I think this'll fail with "argument count mismatch", since argument at L197 wasnt updated

Comment (Brando753) [reply x/ethbridge/types/msgs.go:224]: Great catch Michael. I have updated that section accordingly as well as fixed the unit test for msgs.go

**Comment by Brando753** [comment]
Going to merge this for now, pending review if the prophecy id is needed at all.

**Pull request merged**